extension/js/common/core/mime.ts

```diff
@@ -376,7 +376,20 @@ export class Mime {
   };
 
   private static getNodeAsAttachment = (node: MimeParserNode): Attachment => {
+    let treatAs: 'hidden' | 'encryptedMsg' | undefined;
+    // are we dealing with a PGP/MIME encrypted message?
```

Comment:
@martgil There are two different code paths for parsing messages on gmail page (GmailParser) and on Flowcrypt inbox (here), so there is some code logic duplication. We can think of re-using some common helper function to minimize code duplication.

Reply:
I tried to make a helper function into the `Attachments.ts` based on my understanding. The tests are working fine after code refactoring.

Reply:
Yes, thhat would do. Thank you.